manifests/config.pp

```diff
@@ -181,7 +181,29 @@
       mode    => '0440',
     }
 
-    if ($elasticsearch::version != false and versioncmp($elasticsearch::version, '7.7.0') >= 0) {
+    file { "${elasticsearch::configdir}/jvm.options":
+      ensure  => 'file',
+      notify  => $elasticsearch::_notify_service,
+      require => Class['elasticsearch::package'],
+      owner   => $elasticsearch::elasticsearch_user,
```

Comment:
The service should not require ownership or write permission on configuration files.  It is "safer" to ensure the file belongs to the root user, and that the group the service use has read access to the configuration.

```suggestion
      owner   => 'root',
```